lighthouse-core/audits/accessibility/axe-audit.js

```diff
@@ -42,7 +44,8 @@ class AxeAudit extends Audit {
           type: 'node',
           selector: Array.isArray(node.target) ? node.target.join(' ') : '',
           path: node.path,
-          snippet: node.snippet,
+          snippet: node.html || node.snippet,
+          axeFailureSummary: node.failureSummary,
```

Comment:
how about just `summary`

Reply:
discussed in person `explanation` to match with our audit-level naming 👍